.idea/.name

```diff
@@ -0,0 +1 @@
+IceFlow
```

Comment:
Can you add the `.idea` directory to the `.gitignore` file? This way, local auxiliary files are not being committed.

Reply:
Ok, I just added.